static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.ts

```diff
@@ -624,8 +633,11 @@ describe('DFP', () => {
 	describe('keyword targeting', () => {
 		it('should send page level keywords', async () => {
 			onConsentChange.mockImplementation(
-				(callback: (val: Partial<TCFv2ConsentState>) => void) =>
-					callback(tcfv2WithConsent),
+				(
+					callback: (val: {
+						tcfv2: TCFv2ConsentStateMockType;
```

Comment:
Why do we need to use the Mocked type here?

Reply:
Same as above I think.